css/index.css

```diff
@@ -0,0 +1,125 @@
+
+
+body 
+{background-color: #deb887;
+ text-align: left; /*is this defaut*/
+ font-family: "Oswald", sans-serif;
+ padding: 40px;}
+
+ *
+{margin:2%;}
+
+img 
+{width: 100%;}
+
+h1 
+{font-size: 30px;
+ font-weight: 700;
+ color: maroon;}
+
+h2 
+{font-size: 20px;
+ font-weight: 100;
+ color: maroon}
+
+#about, #liExp, #liProject
+{background-color: bisque;
+ margin: 10px;
+ border:3px solid;
+ border-radius: 10px;
+ padding: 20px;}
+ 
+ 
+
+
+#skills li 
+{background-color: orange;
+ margin: 10px;
+ border: 3px solid;
+ border-radius: 10px;
+ padding: 20px;}
+
+#liEmail
+{background-color: darkkhaki;
+ border: 3px solid;
+ border-radius: 10px;
+ padding: 10px;   
+ margin-bottom: -25px;
+ width: 134px;}
+
+#liGithub
+{background-color: darkkhaki;
+ border: 3px solid;
+ border-radius: 10px;
+ padding: 10px;
+ width: 165px;}
+
+
+#fL1 
+{font-size: 20px;
+ color: red;}
+
+#fL2 
+{font-size: 20px;
+ color: blue;}
+
+#fL3 
+{font-size: 20px;
+ color: green;}
+
+#fI1 
+{font-size: 20px;
+ color:  brown;
+ margin-bottom: 25px;}
+
+#fI2 
+{font-size: 20px;
+ color: brown;
+ margin-bottom: 25px;}
+
+#fI3 
+{font-size: 20px;
+ color: brown;
+ margin-bottom: 25px;}
+
+#formButton
+{color:black;
+ padding-inline:30px;
+ background-color: #D5A021;
+ border-color: #c18e27;}
+
+#leaveMessage
+{background-color:peachpuff;
+padding: 20px;
+border: 3px solid;
+border-radius: 10px;
+margin: 10px;}
+
+button
+{color:black;
+ padding-inline:30px;
+ background-color: #D5A021;
+ border-color: #c18e27;}   
+
+
+
+#foot{
+margin: 10px auto ;
+background-color:goldenrod;
+border: 3px solid;
+border-radius: 10px;
+width: 120px;
+}
+
+@media screen and (max-width: 320px){
+    #fL1,#fL2,#fL3,#fI1,#fI2,#fI3   
+    {margin-left: -30px}
+}
```

Comment:
This seems like it could get out of hand pretty quick as you make changes to these other id's. One option here is to create a class that is only declared in your media query, and apply to each element in your html.

Reply:
done